terraform/cyhy_reporter_cloud_init.tf

```diff
@@ -22,18 +34,39 @@ data "cloudinit_config" "cyhy_reporter_cloud_init_tasks" {
       num_disks     = 2
     })
     content_type = "text/x-shellscript"
-    filename     = "cyhy_reports_disk_setup.sh"
+    filename     = "00_cyhy_reports_disk_setup.sh"
   }
 
   part {
-    content = templatefile("${path.module}/cloud-init/set_hostname.tpl.yml", {
-      # Note that the hostname here is identical to what is set in
-      # the corresponding DNS A record.
-      fqdn     = "reporter.${aws_route53_zone.cyhy_private_zone.name}"
-      hostname = "reporter"
+    content = templatefile("${path.module}/cloud-init/chown_directory.tpl.sh", {
+      group          = "cyhy"
+      is_mount_point = true
+      owner          = "cyhy"
+      path           = "/var/cyhy/reports/output"
     })
-    content_type = "text/cloud-config"
-    filename     = "set_hostname.yml"
-    merge_type   = "list(append)+dict(recurse_array)+str()"
+    content_type = "text/x-shellscript"
+    filename     = "01_cyhy_reports_chown_reports_output_directory.sh"
+  }
+
+  part {
+    content = templatefile("${path.module}/cloud-init/chown_directory.tpl.sh", {
+      group          = "cyhy"
+      is_mount_point = false
+      owner          = "cyhy"
+      path           = "/var/cyhy"
+    })
+    content_type = "text/x-shellscript"
+    filename     = "02_cyhy_reports_chown_cyhy_directory.sh"
```

Comment:
Same question about ordering.

Reply:
I think we are.  We could probably get away with only `chown`ing the parent directory, but it doesn't hurt to do the child directory (and mount point) first.